source/apiVolontaria/volunteer/admin.py

```diff
@@ -88,7 +92,88 @@ class EventAdmin(admin.ModelAdmin):
     ordering = ('start_date',)
 
 
-admin.site.register(models.Cycle)
+def cycle_admin_generate_participation_report(modeladmin, request, queryset):
```

Comment:
Is it possible to put this function inside the admin class ? 
I would just like the file to remain readable with time and new features.